recipe/meta.yaml

```
@@ -15,83 +17,182 @@ source:
     - eigen
     - freetype
-    - python
+    - libboost-devel
+    - libboost-python-devel
     - numpy
+    - pandas
     - pillow
-    - pandas <2.2
     - pip
+    - postgresql
+    - python
     - setuptools
     - setuptools_scm >=8
-    # For rdkit-stubs
-    - pybind11-stubgen
-  run:
-    - cairo
-    - python
-    - pillow
-    - pandas
-    - {{ pin_compatible('numpy') }}
-    - pycairo
-    - matplotlib-base
-    - sqlalchemy
-    - reportlab
-
-test:
-  commands:
-    - python -c "import rdkit; assert rdkit.__version__ == '{{ version }}'"
-  imports:
-    - rdkit
-    - rdkit.Avalon
-    - rdkit.Chem
-    - rdkit.Chem.AllChem
-    - rdkit.Chem.rdFreeSASA
-    - rdkit.DataManip
-    - rdkit.Dbase
-    - rdkit.DistanceGeometry
-    - rdkit.ForceField
-    - rdkit.Geometry
-    - rdkit.ML
-    - rdkit.Numerics
-    - rdkit.SimDivFilters
-    - rdkit.VLib
-    - rdkit.VLib.NodeLib
-    - rdkit.Chem.PandasTools
 
 outputs:
-  - name: rdkit
-  - name: rdkit-dev
-    script: install_rdkit_dev.bat  # [win]
+  - name: librdkit
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
+    build:
+      run_exports:
+        - {{ pin_subpackage('librdkit', exact=True) }}
     requirements:
       build:
         - {{ stdlib("c") }}
+        - cmake
+      host:
+        - libboost-devel
       run:
+        - cairo
+        - eigen
+        - freetype
+    test:
+      commands:
+        - test -f $PREFIX/lib/libRDKitGraphMol.so              # [linux]
+        - test -f $PREFIX/lib/libRDKitGraphMol.dylib           # [osx]
+        - if not exist %LIBRARY_BIN%\RDKitGraphMol.dll exit 1  # [win]
+    about:
+      summary: RDKit C++ library
+      license: BSD-3-Clause
+      license_file: license.txt
+
+  - name: librdkit-dev
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
+    requirements:
+      build:
+        - {{ stdlib("c") }}
+        - cmake
+      run:
+        - {{ pin_subpackage('librdkit', exact=True) }}
+    test:
+      commands:
+        - test -f $PREFIX/include/rdkit/GraphMol/GraphMol.h               # [unix]
+        - if not exist %LIBRARY_INC%\\rdkit\\GraphMol\\GraphMol.h exit 1  # [win]
+    about:
+      summary: RDKit headers and library used in librdkit
+      license: BSD-3-Clause
+      license_file: license.txt
+
+  - name: rdkit
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - {{ stdlib('c') }}
+        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+        - numpy                               # [build_platform != target_platform]
+        - python                              # [build_platform != target_platform]
+        - cmake
+        - jom   # [win]
+        - make  # [unix]
+        - pkg-config
+      host:
+        - cairo
+        - eigen
+        - freetype
+        - libboost-devel
+        - libboost-python-devel
+        - matplotlib-base
```

```suggestion
        - {{ pin_subpackage('librdkit', exact=True) }}
        - matplotlib-base
```

@isuruf gave me the key here, I think. This should make the warnings disappear.

---

my read of the pin_subpackage docs was that this function is only appropriate in run/test deps, but I'll add anyway

---

Done